pyupdater/client/updates.py

```diff
@@ -49,17 +49,12 @@
 
 
 def file_require_admin(file_path):
-    dir_name, exe_file = os.path.split(file_path)
-    cmd = 'cd "{}" && copy "{}" /Y /B +,,'.format(dir_name, exe_file)
-    process = subprocess.Popen(
-        cmd,
-        shell=True,
-        stdin=subprocess.PIPE,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT
-    )
-    output = process.stdout.read().decode("utf-8")
-    return "Access is denied" in output
+    try:
```

Comment:
Made a change passed on the comment from https://github.com/JMSwag/PyUpdater/pull/116#issuecomment-384422747
Need to test if this operates in the same way....

Reply:
Works for me on Python 2 and 3 without special chairs and on Python 3 with special chairs, but not on Python 2 with special chairs. But making the adjustment `with open(file_path.decode('utf-8'), "a"):`  to this works for me on Python 2 with special chairs as well.

Reply:
I'll make that change. How are you testing this?

Reply:
I am testing this by copying the file_require_admin method into a new small python script and calling it for local files on my system.